js/app/cookies-banner.js

```diff
@@ -2,66 +2,47 @@
 var cookiesSet = hasCookiesPreferencesSet();
 var cookiesBanner = $(".js-cookies-banner-form");
```

Comment:
can you rename this to include a $ then it's obvious that you're working with a jquery object
```suggestion
var $cookiesBanner = $(".js-cookies-banner-form");
```

Reply:
As technical lead, I am resolving this as complete.